src/app.ts

```diff
@@ -40,3 +48,92 @@ export default async function newApp(): Promise<Express> {
   }
   return app;
 }
+
+function asyncHandler(func: RequestHandler): RequestHandler {
+  return (req: Request, res: Response, next: NextFunction) =>
+    Promise.resolve(func(req, res, next)).catch(next);
+}
+
+const ipaElasticsearchEndpoint = "https://elasticsearch.developers.italia.it";
```

Comment:
constants should be all caps, unless you plan to read the value from the env (?)